rocAL/rocAL_pybind/example/new_api/rocAL_api_coco_pipeline.py

```diff
@@ -0,0 +1,272 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from math import sqrt
+import torch
+import random
+import itertools
+
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import sys
+import numpy as np
+
+class RALICOCOIterator(object):
+    """
+    COCO RALI iterator for pyTorch.
+    Parameters
+    ----------
+    pipelines : list of amd.rali.pipeline.Pipeline
+                List of pipelines to use
+    size : int
+           Epoch size.
+    """
+
+    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, device="cpu", display=False):
+
+        # self._num_gpus = len(pipelines)
+        assert pipelines is not None, "Number of provided pipelines has to be at least 1"
```

Comment:
the Codacy error is because assert is used without try except. Please change it

Reply:
Changes done in latest commit